js/web/lib/wasm/jsep/webgpu/ops/common.ts

```diff
@@ -785,7 +855,8 @@ class ShaderHelperImpl implements ShaderHelper {
    * Get additional implementation that needs to be added to the shader source.
    */
   get additionalImplementations(): string {
-    return this.uniformDeclaration() + this.indicesHelpers.map(i => i.impl()).join('\n');
+    return this.uniformDeclaration() + this.variables.map(i => i.impl()).join('\n') +
+        this.helpers.map(i => i.impl()).join('\n');
```

Comment:
Does it work if we remove `use` and `helpers` and directly call like below?
```
return this.uniformDeclaration() + this.variables.map(i => i.impl()).join('\n') + this.internalVariables.map(i => i.impl()).join('\n');

Reply:
let me remove 'use'